config.json

```diff
@@ -737,7 +742,14 @@
       "core": true,
       "difficulty": 6,
       "slug": "bank-account",
-      "topics": null,
+      "topics": [
+      	"floating_point_numbers",
+      	"files",
+      	"mathematics",
+      	"logic",
+        "object_oriented_programming",
+      	"variables"
```

Comment:
Thanks for tackling this :) I'm not sure I agree with your suggestions. `floating_point_numbers` and `files` don't appear to be used in this exercise at all. `mathematics`, `logic`, `object_oriented_programming` and `variables` are all used but I wouldn't say they are very important topics. How do you feel they are relevant to the exercise?

I would add `exception_handling` since they tests expect an exception to be thrown, and `concurrency` since that is a very important part of the exercise. You could maybe also add `integers`. What do you think? :)

There are some indentation problems. Each topic should be indented two spaces after `topics`, like `object_oriented_programming`.

Reply:
Alright, I'll come up with more accurate topics and update the PR